actionpack/lib/action_controller/test_case.rb

```diff
@@ -208,10 +207,10 @@ def load!
   end
 
   # Superclass for ActionController functional tests. Functional tests allow you to
-  # test a single controller action per test method. This should not be confused with
-  # integration tests (see ActionDispatch::IntegrationTest), which are more like
-  # "stories" that can involve multiple controllers and multiple actions (i.e. multiple
-  # different HTTP requests).
+  # test a single controller action per test method. Rails discourages use of
+  # ActionController functional tests in favor of integration tests (see ActionDispatch::IntegrationTest).
+  # New Rails applications no longer generate ActionController style functional tests and they should
+  # only be used for backward compatibility. Use ActionDispatch::IntegrationTest instead.
```

Comment:
@kaspth Re. changing `ActionController` to `Action Controller` --> here we are talking about implementation and not framework. `AD::IntegrationTests` are tests for Action Controller as well so saying `Rails discourages use of Action Controller functional tests in favor of integration tests` doesn't sound good to me.